packages/stuffex-frontend/src/UserSellingItems.js

```diff
@@ -4,20 +4,26 @@ import Item from './Item'
 
 function UserSellingItems({ backendRoot }) {
   const [userItems, setUserItems] = useState([])
+  const [user, setUser] = useState({ username: '' })
+
   let { userId } = useParams()
 
   useEffect(() => {
     fetch(`${backendRoot}/users/${userId}/items`)
       .then((res) => res.json())
       .then((data) => setUserItems(data))
       .catch((error) => console.error(error))
+    fetch(`${backendRoot}/users/${userId}`)
+      .then((res) => res.json())
+      .then((data) => setUser(data))
+      .catch((error) => console.error(error))
   }, [userId, backendRoot])
 
   return (
     <Item
       itemData={userItems}
       showSearchBar={false}
-      title="Items You Are Giving Away"
+      title={`Items ${user.username}'Are Giving Away`}
```

Comment:
This looks like it will make something like "Items josh.smithAre Giving Away". It's Friday and we just need to get the PR shipped so it's not a huge issue, but we will need to update this before the project is done.

Reply:
Agreed, we can probably leave the title as is.